lib/ansible/module_utils/openstack.py

```diff
@@ -61,7 +62,7 @@ def openstack_argument_spec():
 def openstack_find_nova_addresses(addresses, ext_tag, key_name=None):
 
     ret = []
-    for (k, v) in addresses.iteritems():
+    for (k, v) in iteritems(addresses):
```

Comment:
@resmo Is there a reason you don't like this?  To work on python3 we either need to do this or:
``` python
for (k, v) in addresses.items():
```

Either way is probably okay.